conf/ref_databases.config

```diff
@@ -16,11 +16,11 @@ params {
       fmtscript = "taxref_reformat_gtdb.sh"
     }
     'pr2=4.13.0' {
-      file = [ "https://github.com/pr2database/pr2database/releases/download/v4.13.0/pr2_version_4.13.0_16S_dada2.fasta.gz" ]
+      file = [ "https://github.com/pr2database/pr2database/releases/download/v4.13.0/pr2_version_4.13.0_18S_dada2.fasta.gz", "https://github.com/pr2database/pr2database/releases/download/v4.13.0/pr2_version_4.13.0_18S_UTAX.fasta.gz" ]
```

Comment:
Probably it would be good to document somewhere that the 18S database is used of pr2 instead of the 16S database? (I admit I am not familiar with pr2)

Actually, probably it would be good to mention all databases and their category in the docs (16S/18S/ITS) to make it easier to choose one. I can do this in a separate PR.

Reply:
Good idea.